### ISSUE TITLE
Erroneous Migrations are being created

### ISSUE THREAD

**Issue by sdc50** [tethysplatform/tethys#610]
When making migrations for Tethys there seems to be something weird going on with the `mfa` migrations:

```
(tethys) $ python tethys_portal/manage.py makemigrations
Loading Tethys Extensions...
Loading Tethys Apps...
Tethys Apps Loaded: helios

Migrations for 'auth':
  $CONDA_HOME/envs/tethys-cpe/lib/python3.7/site-packages/django/contrib/auth/migrations/0012_user_keys.py
    - Create model User_Keys
Migrations for 'mfa':
  $CONDA_HOME/envs/tethys-cpe/lib/python3.7/site-packages/mfa/migrations/0010_delete_user_keys.py
    - Delete model User_Keys
```

It seems that the `User_keys` model is being deleted from the `mfa` migrations and added to the Django migrations. I can't figure out why this is happening. I've attempted to re-create the behavior in a simple environment with just Django and mfa2, but it only seems to happen in the tethys environment.

For now if anyone ends up generating these migrations then I would recommend deleting them before you run the migrations.

**Comment by swainn** [comment]
This might be related to something we did when we implemented MFA. Will take a look.

**Comment by gagelarsen** [comment]
This looks like it has something to do with how we manipulated it to the keys to the user section on the admin pages. Look at where we add it to the tethys_apps admin. There is a function called register_user_keys_admin() where we change the _meta.app_label. That is probably the root of the problem.

https://github.com/tethysplatform/tethys/blob/5ddded97ff32dccceffa4d9e7859994eccf6aeea/tethys_apps/admin.py#L336

**Comment by swainn** [comment]
So it was most likely caused by this change. The problem is we can't commit the needed migrations that are generated in the auth and mfa libraries, which are 3rd party. Or can we? We either need to undo this change or find a way to create commit these migrations with Tethys. What are your thoughts @sdc50?

**Comment by swainn** [comment]
#613 fixes this. However, there is still a missing migration. I believe this one is legitimately missing though. See: https://github.com/mkalioby/django-mfa2/issues/27

**Comment by swainn** [comment]
The remaining migration issue should be fixed in version 2.0.3 of django-mfa2.
